encryption_formats.md

```diff
@@ -181,6 +181,60 @@ None.
 
 Default format for manually created encryption sessions. Used when encrypting file metadata before upload. We reuse the resource ID generated for the file content so that a single resource ID can be reused for multiple encryptions.
 
+### Encryption format v9
+
+#### Spec
+
+| **Element**        | **Buffer type** | **Byte length**   |
+|--------------------| --------------- | ----------------- |
+| Version number (9) | Fixed length    | 1 byte            |
+| Session ID         | Fixed length    | 16 bytes          |
+| Resource ID / Seed | Fixed length    | 16 bytes          |
+| Encrypted data     | Variable length | = clear data size |
+| MAC                | Fixed length    | 16 bytes          |
+
+This format has been designed for simple resource encryption within an encryption session.
+
+#### Properties
+
+Constant overhead: O = 49 bytes
+
+#### Issues
+
+None.
+
+#### Usage
+
+Used instead of format v10 when padding is disabled.
+
+### Encryption format v10
+
+#### Spec
+
+| **Element**         | **Buffer type** | **Byte length**             |
+|---------------------| --------------- |-----------------------------|
+| Version number (10) | Fixed length    | 1 byte                      |
+| Session ID          | Fixed length    | 16 bytes                    |
+| Resource ID / Seed  | Fixed length    | 16 bytes                    |
+| Encrypted data      | Variable length | = clear data size + padding |
+| MAC                 | Fixed length    | 16 bytes                    |
+
+This format has been designed for simple resource encryption within an encryption session.
+It is the same as v8 but with the addition of padding. The padding algorithm and overhead is the same as v6.
+
+#### Properties
+
+Constant overhead: O = 49 bytes
+
+#### Issues
+
+None.
+
+#### Usage
+
+The session used when encrypting with this resource is created automatically (transparent session).
+It is used by default for simple resource encryption, when not using a manually created session.
```

Comment:
* Wording: `with this resource` -> `with this format`?
* Swap sentences for clarity?
* Mention padding?

Reply:
```suggestion
It is used by default for simple resource encryption, when not using a manually created session or disabling padding.
The session used when encrypting with this format is created automatically (transparent session).
```

Reply:
Applied (with slightly different wording for padding, let me know if that looks good)